Client/mods/deathmatch/logic/CStaticFunctionDefinitions.h

```diff
@@ -223,8 +223,9 @@ class CStaticFunctionDefinitions
     static bool            IsVehicleWindowOpen(CClientVehicle& Vehicle, uchar ucWindow);
     static bool            SetVehicleModelExhaustFumesPosition(unsigned short usModel, CVector& vecPosition);
     static bool            GetVehicleModelExhaustFumesPosition(unsigned short usModel, CVector& vecPosition);
-    static bool            SetVehicleModelDummyPosition(unsigned short usModel, eVehicleDummies eDummy, CVector& vecPosition);
-    static bool            GetVehicleModelDummyPosition(unsigned short usModel, eVehicleDummies eDummy, CVector& vecPosition);
+    static bool            SetVehicleModelDummyPosition(unsigned short usModel, eVehicleDummy::e eDummy, CVector& vecPosition);
+    static bool            GetVehicleModelDummyPosition(unsigned short usModel, eVehicleDummy::e eDummy, CVector& vecPosition);
+    static bool            GetVehicleModelDummyPosition(CClientVehicle* pVehicle, unsigned short usModel, eVehicleDummy::e eDummy, CVector& vecPosition);
```

Comment:
Can we change this so that "overload selection" only happens in CLuaDefs, and not in CStaticFunctionDefinitions?

Basically, do whatever is required to make this possible:

```suggestion
    static bool            GetVehicleModelDummyPosition(unsigned short usModel, eVehicleDummy::e eDummy, CVector& vecPosition);
    static bool            GetVehicleModelDummyPosition(CClientVehicle* pVehicle, eVehicleDummy::e eDummy, CVector& vecPosition);
```

That way, in the distant future, if we ever migrate this to the new parser, they can just do `ArgumentParserWarn<OverloadA, OverloadB>` instead of figuring out what's going on here.

If I'm not making sense, I can elaborate.

Reply:
I think I understand what you mean. I'll try to do it.